tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -111,10 +111,21 @@ class TestResourceDeletionDuringMultipleDeleteOperations(ManageTest):
     pvc_size = 3
 
     @pytest.fixture()
-    def setup_base(self, interface, multi_pvc_factory, pod_factory):
+    def setup_base(self, request, interface, multi_pvc_factory, pod_factory):
         """
         Create PVCs and pods
         """
+        if config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS:
+            # Get the index of current consumer cluster
+            self.consumer_cluster_index = config.cur_index
```

Comment:
I'm not sure if I didn't miss anything (maybe in another PR) - but what would happen, if the context will be set to the provider cluster here?